lib/fastlane/plugin/revenuecat_internal/helper/revenuecat_internal_helper.rb

```diff
@@ -28,8 +28,6 @@ def self.replace_version_number(previous_version_number, new_version_number, fil
       def self.edit_changelog(prepopulated_changelog, changelog_latest_path, editor)
         changelog_filename = File.basename(changelog_latest_path)
 
-        UI.user_error!("Pre populated content for changelog was empty") if prepopulated_changelog.empty?
```

Comment:
Hmm I think this was done to avoid making empty releases, but yeah, this is probably not needed. I would suggest outputting a message indicating this though, so changing `user_error!` with `message` or changing the following message to indicate this

Reply:
Good point I'll do that 👍🏻

Reply:
@tonidero done.